css/nav.css

```diff
@@ -1,21 +1,233 @@
 /* nav styling goes here */
 nav {
-    padding-bottom: 14px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    justify-content: flex-end;
+}
+
+nav hr {
+    width: 30%;
+    margin: 1em 0;
 }
 
 nav a {
-    font-weight: bold;
-    padding: 15px 20px;
-    background: white;
-    border-top-left-radius: 20px;
-    border-top-right-radius: 20px;
+    position: relative;
+
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+
     text-decoration: none;
-}
+    font-size: 0.8em;
 
-nav a:not(.active) {
-    background: lightgray;
+    color: white;
 }
 
-nav a:hover:not(.active) {
+nav a:not(#nav-home):hover:not(.active)::before {
     background: grey;
+}
+
+nav a.active {
+    transform: scale(1.1);
+    z-index: 1;
+}
+
+nav a .icon {
+    position: relative;
+    width: 2em;
+    height: 2em;
+    margin-right: 0.5em;
+}
+
+nav a#nav-gacha::before,
+section#gacha-controls {
+    background-color: #3B47B3;
+}
+
+nav a#nav-collection::before,
+section#card-list {
+    background-color: #5F7BC5;
+}
+
+nav a#nav-artist-writer-board::before,
+section#artist-writer-board {
+    background-color: #4C5678;
+}
+
+nav a#nav-message-board::before,
+section#message-board {
+    background-color: #485EB3;
+}
+
+nav a#nav-credits::before,
+section#credits {
+    background-color: #2852B3;
+}
+
+#nav-button {
```

Comment:
(nit) I believe best practice is to have id's be unique. Should this be a class?

Don't worry if this is too tedious to refactor

Reply:
Probably could be named better for sure. It's the hamburger menu button so there's probably only one instance of it. I'll rename it to be something like `#hamburger-nav`